src/cli/services/linter/utils/deduplicateResults.ts

```diff
@@ -1,26 +1,28 @@
-import { IRange } from '@stoplight/types/dist';
+import { resultsComparer } from '../../../../formatters/utils/sortResults';
 import { IRuleResult } from '../../../../types';
 
-type Dictionary<T, K extends PropertyKey> = { [key in K]: T };
+export const deduplicateResults = (results: IRuleResult[]): IRuleResult[] => {
+  const filtered: IRuleResult[] = [];
 
-const ARTIFICIAL_ROOT = Symbol('root');
+  const howMany = results.length;
```

Comment:
```suggestion
  const totalResults = results.length;
```
sounds a tiny bit better to me, but I'm terrible at naming.

Reply:
Fixed